benchmarks/continous/db/tool/cli/src/main.rs

```diff
@@ -0,0 +1,99 @@
+use anyhow::Context;
+use chrono::{DateTime, Utc};
+use clap::{Parser, Subcommand};
+
+use orm::{check_connection, establish_connection, insert_ft_transfer, models::NewFtTransfer};
+
+fn main() -> anyhow::Result<()> {
+    let args = Cli::parse();
+
+    match args.command {
+        Command::CheckConnection => {
+            let connection = &mut establish_connection()?;
+            check_connection(connection)?;
+        }
+        Command::InsertFtTransfer {
+            time,
+            git_commit_hash,
+            git_commit_time,
+            num_nodes,
+            node_hardware,
+            num_traffic_gen_machines,
+            disjoint_workloads,
+            num_shards,
+            num_unique_users,
+            size_state_bytes,
+            tps,
+            total_transactions,
+        } => {
+            let connection = &mut establish_connection()?;
+            let node_hardware: Vec<&str> = node_hardware.iter().map(String::as_str).collect();
+            let new_ft_transfer = NewFtTransfer {
+                time,
+                git_commit_hash: &git_commit_hash,
+                git_commit_time,
+                num_nodes,
+                node_hardware: &node_hardware,
+                num_traffic_gen_machines,
+                disjoint_workloads,
+                num_shards,
+                num_unique_users,
+                size_state_bytes,
+                tps,
+                total_transactions,
+            };
+            insert_ft_transfer(connection, &new_ft_transfer)?;
+        }
+    }
+
+    Ok(())
+}
+
+#[derive(Debug, Parser)]
+#[command(
+    about = "A CLI to interact with the db storing contiuous benchmark data. Commands that connect to the db require the env var DATABASE_URL_CLI to be set in a format compatible with the diesel crate. Consider sourcing the dbprofile file in the repository.",
+    long_about = None
+)]
+struct Cli {
+    #[command(subcommand)]
+    command: Command,
+}
+
+#[derive(Debug, Subcommand)]
+enum Command {
+    /// Connects to the db and runs a SELECT query to check if a connection can be established.
+    CheckConnection,
+    /// Insert data related to an ft transfer benchmark run.
+    #[command(arg_required_else_help = true)]
+    InsertFtTransfer {
```

Comment:
High-level comment: We plan to use this tool in automated fashion and will unlikely ever use it manually after the initial testing. So it makes sense to optimize the interface for automated use.

Concretely, I don't think we need to invest in a user-friendly CLI - the tool can just take JSON as an input, making it easier to both generate it, and parse and pipe it through to the DB.

Reply:
Adressed in 272d649, which makes `insert-ft-transfer` read JSON from a file. The PR description is updated to contain an example.

Just now it occurred to me that in some cases it might be more handy to read a JSON string from stdin. I could add support for that as well.

Reply:
Looks good!

Possibly, but let's not hurry implementing this until we actually need to use it anywhere. The input from file should work fine for the current use-case.